config/initializers/honeycomb.rb

```diff
@@ -34,4 +38,6 @@
       ScholarsArchive::NoiseCancellingSampler.sample(fields)
     end
   end
+else
+   Rails.logger.warn 'Honeycomb disabled, skipping initialization'
```

Comment:
@decimalator We need to do some amount of configuration when `HONEYCOMB_ENABLE` is disabled, otherwise `#append_info_to_payload` gets upset. It can be as simple as
```
  Honeycomb.configure do |config|
  end
```
but idk if there's any other configuration you need to do to silence Honeycomb